### PR TITLE
Clean docs

### PR DESCRIPTION
**Pull request by mdellweg**
Review Checklist:
- [ ] An issue is properly linked. [feature and bugfix only]
- [ ] Tests are present or not feasible.
- [ ] Commits are split in a logical way (not historically).

### CODE REVIEW

**Comment by mdellweg** [comment]
We are not yet ready for this...

File: CHANGES.md [hunk 1]
```diff
@@ -56,8 +55,7 @@
 #### Features {: #0.26.0-feature }
 
 - Added the "pulp rpm prune-packages" command to support new RPM feature.
-
-  See [2909](https://github.com/pulp/pulp_rpm/issues/2909) for details.
+  See [pulp\_rpm#2909](https://github.com/pulp/pulp_rpm/issues/2909) for details.
```
Comment (ggainey) [inline CHANGES.md:58]: :+1:

File: Makefile [hunk 2]
```diff
@@ -40,7 +40,7 @@ docs:
 	pulp-docs build
 
 servedocs:
-	pulp-docs serve -w pulp-glue/pulp_glue -w pulpcore/cli/common/generic.py
+	pulp-docs serve -w CHANGES.md -w pulp-glue/pulp_glue -w pulpcore/cli/common/generic.py
```
Comment (ggainey) [inline Makefile:43]: :+1:

File: pulp-glue/staging_docs [hunk 3]
```diff
@@ -0,0 +1 @@
+docs/
```
Comment (ggainey) [inline pulp-glue/staging_docs:1]: :+1:

File: staging_docs [hunk 4]
```diff
@@ -0,0 +1 @@
+docs
```
Comment (ggainey) [inline staging_docs:1]: :+1:

**Review by ggainey — APPROVED** [review]

**Pull request merged**